### DISCUSSION TITLE
Remix flat route failed to deploy to vercel

### DISCUSSION THREAD

**Discussion by johannchen2021** [vercel/community#1586] [Help] [answered]
### Summary

After I configured my remix app with it's flat route naming conventions (https://remix.run/docs/en/v1/file-conventions/route-files-v2), it works locally on my dev machine but it failed to deploy to Vercel. It has the following error message during the build in Vercel. 

`Error: File not found: /vercel/path0/app/outes/_index.tsx`

Please help

btw, the path seems off.  'outes' should be 'routes'. the vercel build tool somehow cut a letter?

**Comment by TooTallNate** [comment, marked as answer]
Are you still experiencing this issue? This should have been fixed by https://github.com/vercel/vercel/pull/9519.

**Reply by johannchen2021** [reply]
Thanks, it works now!
